src/services/tx/txSender.ts

```diff
@@ -72,6 +72,9 @@ const getRecommendedTxParams = async (
   const safeSDK = getAndValidateSafeSDK()
   const chainId = await safeSDK.getChainId()
   const safeAddress = safeSDK.getAddress()
+  const contractVersion = await safeSDK.getContractVersion()
+  const safeTxGasRequired = isSafeTxGasRequired(contractVersion)
```

Comment:
I would rewrite this as follows, but not a big deal
```js
  const isSafeTxGasRequired = isLegacyVersion(contractVersion)
```

Reply:
done señor